clang/lib/CIR/Dialect/Transforms/TargetLowering/LowerFunction.cpp

```diff
@@ -519,7 +846,39 @@ Value LowerFunction::rewriteCallOp(const LowerFunctionInfo &CallInfo,
         }
       }
 
-      llvm_unreachable("NYI");
+      // If coercing a fixed vector from a scalable vector for ABI
+      // compatibility, and the types match, use the llvm.vector.extract
+      // intrinsic to perform the conversion.
+      if (::cir::MissingFeatures::vectorType()) {
+        llvm_unreachable("NYI");
+      }
+
+      // FIXME(cir): Use return value slot here.
```

Comment:
missing features here

Reply:
Not a feature. It's an implementation detail.